packages/arb-token-bridge-ui/src/util/AnalyticsUtils.ts

```diff
@@ -19,6 +19,16 @@ export type ProviderName =
   | 'Other'
 
 type AnalyticsEventMap = {
+  'Transfer Button Clicked': {
```

Comment:
Let's call it 'Transfer Button Click' so it's consistent with all of the other click event names.